source/codegen/templates/service.cpp.mako

```diff
@@ -51,6 +51,9 @@ resource_repository_deps = service_helpers.get_driver_shared_resource_repository
 % endif
 % if any_non_mockable_functions:
 #include "${module_name}_library.h"
+#ifdef __GNUC__
```

Comment:
Move this below the last `#include` so it only affects this file, not the `#include`.